mlbench_core/evaluation/goals.py

```diff
@@ -113,7 +122,67 @@ def task3_time_to_preplexity_light_goal(metric_name, value, tracker):
 
     if value <= 100:
         duration = tracker.get_total_train_time()
-        result = "Validation perplexity of 50 reached in {0:.3f} seconds"\
+        result = "Validation perplexity of 50 reached in {0:.3f} seconds" \
+            .format(duration)
+
+        result = _add_detailed_times(result, tracker)
+
+        return result
+
+    return None
+
+
+def task2_time_to_accuracy_goal(metric_name, value, tracker):
+    """Time to perplexity goal for benchmark task 2: Linear binary classifier
+
+    Target is an accuracy of 89%
+
+    Args:
+        metric_name(str): Name of the metric to test the value for, only
+            "val_global_Accuracy@0" is counted
```

Comment:
The other goal call this `Prec` for precision, but it's essentially the same as accuracy. We should keep naming consistent across goals (I personally prefer `Accuracy`, as `Prec` is not that common)

Reply:
indeed, accuracy is more common, and should be enough for the epsilon task (in addition to recording the training loss as well)

Reply:
Then maybe we should unify both of these goal functions and parametrize the goal accuracy ?